# Take MBROLA voices form MBROLA EasyBuild package
# SEP 8th 2020 by B. Hajgato (UGent)
--- src/libespeak-ng/synth_mbrola.c.orig	2019-11-29 17:50:54.000000000 +0100
+++ src/libespeak-ng/synth_mbrola.c	2020-09-08 17:58:01.806240614 +0200
@@ -90,7 +90,7 @@
 	// if not found, then also look in
 	//   usr/share/mbrola/xx, /usr/share/mbrola/xx/xx, /usr/share/mbrola/voices/xx
 	if (GetFileLength(path) <= 0) {
-		sprintf(path, "/usr/share/mbrola/%s", mbrola_voice);
+		sprintf(path, "%s/share/mbrola/%s/%s", getenv("EBROOTMBROLA"), mbrola_voice, mbrola_voice);
 
 		if (GetFileLength(path) <= 0) {
 			sprintf(path, "/usr/share/mbrola/%s/%s", mbrola_voice, mbrola_voice);
